beta-src/src/components/ui/WDMessage.tsx

```diff
@@ -33,24 +37,22 @@ const WDMessage: React.FC<WDMessageProps> = function ({
   const msgWidth = mobileLandscapeLayout ? "170px" : "250px";
   const justify =
     userCountry && message.fromCountryID === userCountry.countryID
-      ? "flex-end"
-      : "flex";
+      ? "end"
+      : "start";
   const msgTime = new Date(0);
   msgTime.setUTCSeconds(message.timeSent);
   return (
-    <Box sx={{ display: "flex", justifyContent: justify }}>
-      <Box
+    <div className={`flex justify-${justify}`}>
+      <div
         id={`message-${String(message.timeSent)}`}
```

Comment:
You don't need this `String()` call here. Interpolated strings are already coerced.